webapp/src/main/java/com/box/l10n/mojito/entity/security/user/User.java

```diff
@@ -55,6 +55,9 @@ public class User extends AuditableEntity implements Serializable {
   @JsonView(View.IdAndName.class)
   String commonName;
 
+  @Column(name = "can_translate_all_locales", nullable = false)
+  boolean canTranslateAllLocales = true;
```

Comment:
I think, we should keep it as `true`. The default role for new users is `USER`, so it will likely be fairly common to accidentally create `USER` users instead of `TRANSLATOR` users. If the `ADMIN` then changes the role for the new user, then the new user will still be unable to translate anything, because the `canTranslateAllLocales` is now `false`. --> The `ADMIN` now has to change two settings instead of one.

Additionally, in your scenario, what is supposed to happen if the `ADMIN` downgrades a `TRANSLATOR` to `USER`? Should `canTranslateAllLocales` also be disabled? In this case the role selection dropdown would effectively control two inputs, which feels inconsistent and unintuitive. If the `canTranslateAllLocales` field is not disabled it is inconsistent with creating a new user.

---

So, I would rather solve this in the GUI, by disabling the checkbox for `USER` and displaying a short note explaining that this option won't have an effect because of the currently selected role. IMHO, this would have the least surprising behaviour for all users.

Reply:
I'm happy for it to be handled in the GUI as you describe 👍

Reply:
Should be implemented now.